job_scripts/cori-haswell/chainslurm.sh

```diff
@@ -25,10 +28,25 @@ if [ $numjobs -gt "20" ]; then
     exit -1
 fi
 
-echo chaining $numjobs jobs starting with $oldjob
-
+firstcount=1
+
+if [ $oldjob -eq "-1" ]
+then 
+    echo chaining $numjobs jobs
+    
+    echo starting job 1 with no dependency
+    aout=`sbatch ${script}`
+    id=$(echo $aout | cut -c21-28)
```

Comment:
I think it will be cleaner (and more robust to changes) to launch the job with `--parsable`` as we do below -- then the job id is the returned output

Reply:
Good idea. I added and tested the parsable flag.